tests/php-mode-test.el

```diff
@@ -659,6 +659,7 @@ Meant for `php-mode-test-issue-503'."
 
 (ert-deftest php-mode-test-php81 ()
   "Test highlighting language constructs added in PHP 8.1."
+  (skip-unless (executable-find "php8.1"))
```

Comment:
Why skip this test when the `php8.1` exec doesn't exist?

PHP Mode highlights don't depend on the PHP executable, so the tests have actually passed on both my local PC and [GitHub Actions](https://github.com/emacs-php/php-mode/blob/master/.github/workflows/test.yml).